iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractConnectableBranch.java

```diff
@@ -102,6 +102,16 @@ public CurrentLimitsAdder newCurrentLimits1() {
         return getOperationalLimitsHolder1().newCurrentLimits();
```

Comment:
maybe do not use deprecated method here and where Sonar is asking but the new one to avoid sonar issues on maintainability?
```suggestion
        return getOperationalLimitsHolder1().getOrCreateSelectedOperationalLimitsGroup().newCurrentLimits();
```